chroma-manager/tests/integration/core/remote_operations.py

```diff
@@ -1143,7 +1143,22 @@ def has_chroma_agent(self, server):
     def stop_agents(self, server_list):
         for server in server_list:
             if self.has_chroma_agent(server):
-                self._ssh_address(server, 'service chroma-agent stop')
+                self._ssh_address(
+                    server, 
```

Comment:
I have a pep8 / pyflakes auto-formatter in my editor, but I turn it off because it makes numerous changes. Probably worth biting the bullet and formatting this whole repo.